src/datasets/dto/output-dataset-obsolete.dto.ts

```diff
@@ -92,14 +92,14 @@ export class OutputDatasetObsoleteDto extends UpdateDatasetObsoleteDto {
   @IsString()
   readonly dataFormat?: string;
 
-  @ApiProperty({
-    type: String,
-    required: false,
-    description: "The ID of the proposal to which the dataset belongs.",
-  })
-  @IsOptional()
-  @IsString()
-  readonly proposalId?: string;
+  // @ApiProperty({
+  //   type: String,
+  //   required: false,
+  //   description: "The ID of the proposal to which the dataset belongs.",
+  // })
+  // @IsOptional()
+  // @IsString()
+  // readonly proposalId?: string;
```

Comment:
Why is proposalId commented out from the dto? Does it mean we no longer provide proposalId for retrieved dataset?

Reply:
I transfer the field in the output dataset obsolete and I forgot to delete it.